data-structures-linked-lists/append.js

```diff
@@ -0,0 +1,15 @@
+/* global LinkedList */ // eslint-disable-line no-unused-vars
+/* exported append */
+
+function append(list, value) {
+  const newList = new LinkedList(value);
+  if (list.next === null) {
+    list = newList;
+    return list;
+  }
```

Comment:
I think you can actually remove this whole `if` statement? seems a little confusing